tasks/apply-mapping/tests/test-apply-mapping.yaml

```diff
@@ -125,6 +125,10 @@ spec:
               test $(cat $(workspaces.config.path)/test_snapshot_spec.json \
                 | jq -r '.components[] | select(.name=="comp4") | .repository') == repo4
 
+              echo Test that customfield of component comp4 was added to its snapshot entry
```

Comment:
Just curious, how did you think of this? What made you add this now? And what's the use case for having a custom field anyway?

Reply:
I wanted to make sure `pushSourceContainer: true` on a component was propagated to the resulting snapshot_spec file

Reply:
ok, thanks